README.md

```diff
@@ -0,0 +1,11 @@
+# bitcoin-price-demo
```

Comment:
I think you are doing this at present, but if not add an install section which documents the commands to run (i.e., `npm install; bower install`)